server/src/main/scala/org/http4s/server/middleware/Logger.scala

```diff
@@ -46,6 +62,14 @@ object Logger {
   )(httpApp: HttpApp[F]): HttpApp[F] =
     apply(logHeaders, logBody, FunctionK.id[F], redactHeadersWhen, logAction)(httpApp)
 
+  def httpAppLogBodyText[F[_]: Concurrent](
```

Comment:
A lot of the middlewares with overloads are introducing an `httpApp` and `httpRoutes` object, and then putting the overloads in there.  What's currently `httpApp` would be the `apply` in that object, and then this would be `logBodyText` in that object.

Reply:
To make sure that I follow, @rossabaker, you're suggesting that I re-name `httpAppLogBodyText` and `httpRoutesLogBodyText` to `httpApp` and `httpRoutes`, respectively, as overloads?

Reply:
More like

```scala
object httpApp {
  def apply(...)
  def logBodyText(...)
}
```

Reply:
Thanks, Ross. That makes sense! I'll make those changes.

Reply:
Please let me know, @rossabaker, if https://github.com/http4s/http4s/pull/3497#commits-pushed-a3c6f32 addresses your comment.

Reply:
If I understand the build output properly, my change may not be binary compatible:

```
[error] server: Failed binary compatibility check against org.http4s:http4s-server_2.12:0.21.0! Found 36 potential problems
[error]  * method httpApp(Boolean,Boolean,scala.Function1,scala.Option,cats.data.Kleisli,cats.effect.Concurrent)cats.data.Kleisli in object org.http4s.server.middleware.ResponseLogger does not have a correspondent in current version
```

Reply:
Ugh.  I didn't think of that.  MiMa is right.  I guess we've never added an overload since introducing this pattern.  For 0.21, let's just chop it off at 32af97d.  Sorry about that.